src/Tests/Seam/When_dispatching_messages_in_receive_context.cs

```diff
@@ -62,7 +62,7 @@ public class When_dispatching_messages_in_receive_context
                 var bytes = Encoding.UTF8.GetBytes("Whatever");
                 var outgoingMessage = new OutgoingMessage("Id-1", new Dictionary<string, string>(), bytes);
 
-                var ctx = context.Context.Get<ReceiveContext>();
+                var ctx = context.Context.Get<ReceiveContextAwareTransaction>().ReceiveContext;
```

Comment:
The `ReceiveContext` is not longer in the `ContextBag`, but in the `TransportTransaction`.